cmd/helm/template_test.go

```diff
@@ -131,6 +131,12 @@ func TestTemplateCmd(t *testing.T) {
 			cmd:    fmt.Sprintf(`template '%s' --skip-tests`, chartPath),
 			golden: "output/template-skip-tests.txt",
 		},
+		{
+			name:      "template with required statement",
```

Comment:
Maybe rename this test to reflect the root cause as you stated here: https://github.com/helm/helm/issues/5550#issuecomment-879130623.